R/infrastructure-git.R

```diff
@@ -98,3 +150,36 @@ add_git_ignore <- function(pkg = ".", ignores) {
   invisible(TRUE)
 }
 
+#' Add GitHub links to DESCRIPTION.
+#'
+#' Populates the URL and BugReports fields of DESCRIPTION with
+#' \code{https://github.com/<USERNAME>/<REPO>} AND
+#' \code{https://github.com/<USERNAME>/<REPO>/issues}, respectively. If package
+#' does not already use GitHub, the above links will be used quite literally.
+#'
+#' @inheritParams use_git
+#' @family git infrastructure
+#' @keywords internal
+#' @export
+use_github_links <- function(pkg = ".") {
```

Comment:
The uses functions are normally designed to fail instead of overriding existing fields